src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/collectionsgrid/view/CollectionsGrid.kt

```diff
@@ -0,0 +1,83 @@
+package org.wycliffeassociates.otter.jvm.app.ui.collectionsgrid.view
+
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.beans.property.Property
+import javafx.scene.layout.Priority
+import javafx.stage.Screen
+import org.wycliffeassociates.otter.common.data.model.Collection
+import org.wycliffeassociates.otter.jvm.app.theme.AppStyles
+import org.wycliffeassociates.otter.jvm.app.theme.AppTheme
+import org.wycliffeassociates.otter.jvm.app.ui.collectionsgrid.CollectionGridStyles
+import org.wycliffeassociates.otter.jvm.app.ui.collectionsgrid.viewmodel.ViewCollectionsViewModel
+import org.wycliffeassociates.otter.jvm.app.widgets.card.DefaultStyles
+import org.wycliffeassociates.otter.jvm.app.widgets.card.card
+import tornadofx.*
+
+class CollectionsGrid : Fragment() {
+    private val viewModel: ViewCollectionsViewModel by inject()
+
+    val activeCollection: Property<Collection> = viewModel.activeCollectionProperty
+    val activeProject: Property<Collection> = viewModel.activeProjectProperty
+
+    init {
+        importStylesheet<CollectionGridStyles>()
+        importStylesheet<DefaultStyles>()
+    }
+
+    override val root = stackpane {
+        hgrow = Priority.ALWAYS
+        vgrow = Priority.ALWAYS
+        style {
+            prefWidth = Screen.getPrimary().visualBounds.width.px - 50.0
+            prefHeight = Screen.getPrimary().visualBounds.height.px - 50.0
+        }
+        addClass(AppStyles.appBackground)
+        hbox {
+
+            vbox {
+                hgrow = Priority.ALWAYS
+                vbox {
+                    vgrow = Priority.ALWAYS
+                    progressindicator {
+                        visibleProperty().bind(viewModel.loadingProperty)
+                        managedProperty().bind(visibleProperty())
+                        addClass(CollectionGridStyles.contentLoadingProgress)
+                    }
+                    scrollpane {
+                        isFitToHeight = true
+                        isFitToWidth = true
+                        flowpane {
```

Comment:
One thing I'd like to assign you (and we can set this up as a spike and list it as a user story) is to do some compare and contrasting with flowpane and listview; Compare/contrast, but also I want to know the performance impact on both.

Doesn't necessarily need to be a formal report, just informally let me know what you find